benchmarks/src/gatling/resources/gatling.conf

```diff
@@ -0,0 +1,10 @@
+gatling {
+  charting {
+    indicators {
+      percentile1 = 25
+      percentile2 = 50
+      percentile3 = 75
```

Comment:
IQR :tada: ;)

Reply:
Yes that one is specifically for your box plots :-)